README.md

```diff
@@ -1 +1,43 @@
 # python-esileapclient
+
+Python API for interacting with ESI-Leap
+
+###Overview
+
+This is a client for the OpenStack Lease API. It provides:
+
+   - a openstack command-line plugin: openstack lease
+
+python-esileapclient is licensed under the Apache License, Version 2.0, like the rest of OpenStack.
+
+###openstack lease CLI
+
+The openstack lease command-line interface is available when the lease plugin (included in this package) is used with the `OpenStackClient https://docs.openstack.org/python-openstackclient/latest/
+
+The client uses keystone to look for the lease service and currently does not allow for endpoint overriding.
+
+To install this package,
+ - clone this repository
+ - install all requirements:  `pip install -r requirements.txt`
+ - install:    `python setup.py install`
+
+Examples:
+
+    openstack lease offer list
+
+will make a GET request to ESI-Leap and print to screen a list of all the offers in the ESI-Leap database.
+
+    openstack lease offer show 'uuid'
```

Comment:
Use `<uuid>` to indicate a parameter whose value should be supplied by the user. Excellent documentation otherwise!